examples/martini-example/main.go

```diff
@@ -53,5 +51,6 @@ func PingHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func SecuredPingHandler(w http.ResponseWriter, r *http.Request) {
+	// TODO(jayhelton) the martini example doesnt seem to pass along the middleware on this branch, it receives and <invalid Value> message and skips the handler
 	respondJSON("All good. You only get this message if you're authenticated", w)
```

Comment:
This was a weird issue. Ill have to debug through this one more to see if its the branch updates